src/components/RecipeEdit.vue

```diff
@@ -251,6 +276,7 @@ export default {
             }
         },
         setup: function() {
+            let $this = this
```

Comment:
May I ask, mainly out of curiosity, why we need a `$this` variable when we have the `this` one?

Reply:
Good spot :o I introduced it for the $nextTick callback. But actually it’s not necessary since $nextTick is invoked in the `this` context